fedmsg/commands/trigger.py

```diff
@@ -110,11 +110,11 @@ def run(self):
         max_queue_size = int(self.config['max_queue_size'])
 
         timer = None
-        queue = Queue.Queue()
+        que = queue.Queue()
```

Comment:
I decided to do this because I consider renaming imports misleading. When we finally fully switch to Python 3, there will be no `Queue` import, just `queue`.
I find it usual that people use variables named `klass` or so, so that's why I chose `que`, which sounds pretty much like `queue` when I read it aloud.